package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "webpagetest",
-  "version": "0.5.0",
+  "version": "0.5.1",
```

Comment:
Actually, let's not bump the version yet, @mbrevda. I'll throw in one other param we wanted to add and then bump the version.